### ISSUE TITLE
Ajenti First Start Ubuntu Server 22.04

### ISSUE THREAD

**Issue by beigeninja** [ajenti/ajenti#1412]
Hi Please may I have some help with this error. First time installing Ajenti, following the auto install process. Getting the following error after reboot. 

`<pre> ajenti.service - Ajenti panel
     Loaded: loaded (/lib/systemd/system/ajenti.service; enabled; vendor preset: enabled)
     Active: <font color="#CC0000"><b>failed</b></font> (Result: exit-code) since Sat 2022-11-05 10:34:06 SAST; 7min ago
    Process: 137265 ExecStart=/usr/bin/python3 /usr/local/bin/ajenti-panel -d <font color="#CC0000"><b>(code=exited, </b></font><span style="background-color:#F2F2F2"><font color="#333333">&gt;</font></span>
        CPU: 430ms

Nov 05 10:34:04 systemd[1]: Starting Ajenti panel...
Nov 05 10:34:05 python3[137265]: Traceback (most recent call last):
Nov 05 10:34:05 python3[137265]:   File &quot;/usr/local/bin/ajenti-panel&quot;, line 123, i<span style="background-color:#F2F2F2"><font color="#333333">&gt;</font></span>
Nov 05 10:34:05 python3[137265]:     if not os.path.exists(config_path):
Nov 05 10:34:05 python3[137265]:   File &quot;/usr/lib/python3.10/genericpath.py&quot;, line<span style="background-color:#F2F2F2"><font color="#333333">&gt;</font></span>
Nov 05 10:34:05 python3[137265]:     os.stat(path)
Nov 05 10:34:05 python3[137265]: TypeError: stat: path should be string, bytes, os<span style="background-color:#F2F2F2"><font color="#333333">&gt;</font></span>
Nov 05 10:34:06 systemd[1]: <b>ajenti.service: Control process exited, code=exited, s</b><span style="background-color:#F2F2F2"><font color="#333333">&gt;</font></span>
Nov 05 10:34:06  systemd[1]: <font color="#D7D75F"><b>ajenti.service: Failed with result &apos;exit-code&apos;.</b></font>
Nov 05 10:34:06 systemd[1]: <font color="#CC0000"><b>Failed to start Ajenti panel.</b></font>
<span style="background-color:#F2F2F2"><font color="#333333">lines 1-16/16 (END)</font></span>
</pre>`

**Comment by kiarn** [comment]
Hello @beigeninja 

Sorry for the late reply. Did you have an error message during the install process ?
It seems that the post install script did not create the config file, and it would interesting to understand why.

You can manually solve it with : 
- creating a file `/etc/ajenti/config.yml` with the content : 

```
auth:
  allow_sudo: true
  emails: {}
  provider: os
  users_file: /etc/ajenti/users.yml
bind:
  host: 0.0.0.0
  mode: tcp
  port: 8000
color: default
max_sessions: 9
trusted_domains: []
trusted_proxies: []
session_max_time: 3600
name: YOUR_HOSTNAME
ssl:
  certificate:
  fqdn_certificate:
  force: false
  client_auth:
    certificates: []
    enable: false
    force: false
  enable: false
```

- then creating a file `/etc/ajenti/users.yml` containing `users: null`
- then launching `ajenti-ssl-gen $(hostname)`
- and finally restarting `systemctl restart ajenti`

Arnaud

**Comment by beigeninja** [comment]
@kiarn Thanks for the response. I did have a few errors trying to install. Specifically I found several different install guides similar in nature but some missing a few steps. So I ended trying the manual install then removed and reinstalled using aut script. Let me try again also following your suggestion and record any issues as I go along. Hopefully it helps identify any bugs for your future releases. I appreciate the help thank you.
